scss/_settings_colors.scss

```diff
@@ -226,6 +226,9 @@ $colors--theme--border-default: var(--vf-color-border-default);
 $colors--theme--border-high-contrast: var(--vf-color-border-high-contrast);
 $colors--theme--border-low-contrast: var(--vf-color-border-low-contrast);
 
+$color--theme-positive: var(--vf-color-positive);
```

Comment:
Rename those to `$color--theme--positive` and `--negative` (double dash after theme, plz.
Also add caution and information colours, that seems to be missing now from the theming.

Even if they don't have different theme versions at the moment, please add them to theme variables (with same value in both themes).